src/lib/Libutil/misc_utils.c

```diff
@@ -1835,9 +1837,16 @@ create_query_file(void)
 {
 	FILE *f;
 	char filename[MAXPATHLEN + 1];
+#ifdef WIN32
+	LPSTR win_sid = NULL;
+	ConvertSidToStringSid(getuid(), &win_sid);
```

Comment:
Do `LocalFree` on win_sid after use. Also, check for a return value of `ConvertSidToStringSid` and `getuid`